sktime/forecasting/adapters/_autots.py

```diff
@@ -0,0 +1,449 @@
+from __future__ import annotations
+
+__author__ = ["MBristle"]
+
+import logging
+
+import pandas as pd
+
+from sktime.forecasting.base import BaseForecaster, ForecastingHorizon
+
+
+class AutoTSAdapter(BaseForecaster):
+    """Act as adapter for the AutoTS library.
+
+    Parameter:
+    ---------
+        fh (ForecastingHorizon): The forecasting horizon.
+        model_list (str): The list of models to use. See docs for details.
+        frequency (str): The frequency of the time series.
+        prediction_interval (float): The prediction interval for the forecast.
+        max_generations (int): The maximum number of
+            generations for the genetic algorithm.
+        no_negatives (bool): Whether negative values are allowed in the forecast.
+        constraint (float): The constraint on the forecast values.
+        ensemble (str): The ensemble method to use.
+        initial_template (str): The initial template to use for the forecast.
+        random_seed (int): The random seed for reproducibility.
+        holiday_country (str): The country for holiday effects.
+        subset (int): The subset of the data to use.
+        aggfunc (str): The aggregation function to use.
+        na_tolerance (float): The tolerance for missing values.
+        metric_weighting (dict): The weights for different forecast evaluation metrics.
+        drop_most_recent (int): The number of most recent data points to drop.
+        drop_data_older_than_periods (int): The threshold for dropping old data points.
+        transformer_list (dict): The list of transformers to use.
+        transformer_max_depth (int): The maximum depth of the transformers.
+        models_mode (str): The mode for selecting models.
+        num_validations (int): The number of validations to perform.
+        models_to_validate (float): The fraction of models to validate.
+        max_per_model_class (int): The maximum number of models per class.
+        validation_method (str): The method for validation.
+        min_allowed_train_percent (float): The minimum percentage of
+            data allowed for training.
+        remove_leading_zeroes (bool): Whether to remove leading zeroes from the data.
+        prefill_na (str): The method for prefilling missing values.
+        introduce_na (bool): Whether to introduce missing values to the data.
+        preclean (dict): The parameters for data pre-cleaning.
+        model_interrupt (bool): Whether the model can be interrupted.
+        generation_timeout (int): The timeout for each generation.
+        current_model_file (str): The file containing the current model.
+        verbose (int): The verbosity level.
+        n_jobs (int): The number of jobs to run in parallel.
+        model_name: (str): The name of the model.
+            NOTE: Overwrites the model_list parameter. For using only one model.
+    """
+
+    _tags = {
+        "scitype:y": "univariate",
```

Comment:
Can it not be multivariate as well, depending value passed to `model_list`?

Reply:
hm, that seems obscure from the source documentation...

Reply:
I have not tested or used it with multivariate datasets and, therefore, have not tried it so far. Maybe this is another enhancement (with the exogenous variables and the probabilistic forecasts).

Reply:
Ref. https://winedarksea.github.io/AutoTS/build/html/source/tutorial.html#id4

> By default, most available models are tried. For a more limited subset of models, a custom list can be passed in, or more simply, a string, one of 'probabilistic', 'multivariate', 'fast', 'superfast', or 'all'.

I think there is a chance user may take values from here and pass. If it is not possible support all of these, at least not to start with, may be add a note on what all is currently supported?